core/embed/projects/bootloader/main.c

```diff
@@ -91,9 +94,13 @@ void failed_jump_to_firmware(void);
 CONFIDENTIAL volatile secbool dont_optimize_out_true = sectrue;
 CONFIDENTIAL void (*volatile firmware_jump_fn)(void) = failed_jump_to_firmware;
 
-static secbool is_manufacturing_mode(void) {
+static secbool is_manufacturing_mode(vendor_header *vhdr) {
   unit_properties_init();
 
+  if ((vhdr->vtrust & VTRUST_ALLOW_PROVISIONING) == VTRUST_ALLOW_PROVISIONING) {
```

Comment:
what does the `is_manufacturing_mode` function mean?

because this reads like the inverse: if provisioning is allowed, then manufacturing mode is false?

Reply:
yes that is a bug, thanks https://github.com/trezor/trezor-firmware/commit/fbde08d505f84415b0bf38a6657b614b7acb1fd3